template_example/dev/vagrant/bootstrap/README.md

```diff
@@ -0,0 +1,15 @@
+# Customize the startup procedure of the Hashistack with pre- and post start ansible scripts
+
+You may put any number of script files in this directory for running ansible commands prior to bootstrapping the hashistack.
+The bootstrap procedure is included/hardcoded in your box.
```

Comment:
```suggestion
The bootstrap procedure is included/hardcoded in vagrant-hashistack box.
```

Reply:
These should probably  be fixes in the template aswell